### ISSUE TITLE
Is this project still being supported?

### ISSUE THREAD

**Issue by pfielding** [MisterWil/abodepy#86]
Just wondering if anyone is still doing work on this project, as the last update seems to be some time ago.  I've been using it successfully for quite some time, and saw that Abode has just gone to 6.0 of their software - wondering if anyone has tried updating successfully without it borking abodepy?

**Comment by kevdliu** [comment]
Abode 6.0 is mostly UI changes so abodepy is not affected. I updated yesterday and abodepy is still working fine.

**Comment by jaraco** [comment]
According to https://github.com/MisterWil/abodepy/pull/85#issuecomment-1140621413, the owner of this repo is no longer active on the project. Probably that means it's abandoned.

I'm interested in supporting this project.

**Comment by kevdliu** [comment]
@jaraco Lmk if you end up forking and running with the project. I have a few minor fixes on my fork that I didn't bother creating PR's for since the owner is no longer active.

**Comment by pfielding** [comment]
Yes please do keep this alive.  I certainly have an interest in this.  I don’t have the skill to do serious writing, but I am reasonably good at testing problem solving….

**Comment by jaraco** [comment]
I've published [jaraco.abode 0.1](https://pypi.org/project/jaraco.abode). It's currently a drop-in replacement for abodepy 1.2.1. I plan to do a lot more work to migrate the project to its own namespace.
